### PR TITLE
support with-bytes feature for protobuf

### PR DESCRIPTION
**Pull request by hicqu**
Signed-off-by: qupeng <qupeng@pingcap.com>

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -10,7 +10,7 @@ description = "Utility functions for generating Rust code from protobufs (using
 
 [features]
 default = ["protobuf-codec"]
-protobuf-codec = ["protobuf-codegen", "protobuf", "regex"]
+protobuf-codec = ["protobuf-codegen", "protobuf/with-bytes", "regex"]
```
Comment (BusyJay) [inline Cargo.toml:13]: Is the feature necessary?

File: src/protobuf_impl.rs [hunk 2]
```diff
@@ -103,7 +103,11 @@ impl Builder {
             desc.get_file(),
             &files_to_generate,
             &Path::new(&self.out_dir),
-            &protobuf_codegen::Customize::default(),
+            &protobuf_codegen::Customize {
+                carllerche_bytes_for_bytes: Some(true),
```
Comment (BusyJay) [inline src/protobuf_impl.rs]: How about using extension instead?

**Review by nrc — APPROVED** [review]
lgtm

File: src/protobuf_impl.rs (hunk 2 above)
Comment (hicqu) [reply src/protobuf_impl.rs]: `lite_runtime` is a different option in `Customize`. Here is the code:
```rust
let lite_runtime = rustproto::exts::lite_runtime_all.get(source);
Customize {
    lite_runtime,
    // carllerche_bytes_for_bytes is not set.
    ..Default::default(),
}
```

File: Cargo.toml (hunk 1 above)
Comment (hicqu) [reply Cargo.toml:13]: Yes, otherwise some function will can't be found.

**Review by BusyJay — APPROVED** [review]

**Pull request merged**